PsySys/vignettes/my-vignette.Rmd

```diff
@@ -0,0 +1,116 @@
+
+## The Rationale behind PsySys 
+
+Over the past decade, the network approach to psychopathology has gained considerable attention within the scientific community as an alternative way to conceptualize and study mental disorders (see Robinaugh et al. (2020) for a review). In contrast to the predominant common cause model postulating that a given disorder causes a set of observed symptoms, the network approach focuses on the dynamic interplay between the symptoms that is thought to give rise to, and maintain, the disorder (Borsboom, 2008). Thereby, the network approach breaks down clear disorder boundaries, which has proven fruitful in better understanding not only the heterogeneous structure but also the observed co-morbidity in various mental disorders (Fried et al., 2017).
+
+Most applications of the network approach in practice have relied on the estimation and analysis of personalized networks, aimed at reforming patient-centered care (Burger et al., 2020). However, the considerable limitations of the corresponding statistical methods, mainly linked to the extensive data they require, result in a high participant burden, and thereby complicate their applicability. An alternative method to examine symptom relations has been proposed with Perceived 
+Causal Relationship (PCR) scaling (Frewen, Allen, Lanius, & Neufeld, 2012). Instead of taking on a data-driven approach, PCR scaling relies on the individuals perceptions to create symptom networks. After identifying the set of symptoms they are currently dealing with, respondents go through each symptom pair $s_{i,j}$ to indicate the perceived relations between them , thereby iteratively building their own network. While the PCR method offers a promising alternative to data-driven network estimation techniques, existing clinically adapted versions of the method, such as the PECAN questionnaire (Klintwall et al., 2021), are characterized by a repetitive structure which has been met with increasing dissatisfaction among respondents. Thus, there is the need to explore alternative, more user-friendly, PCR implementations to improve its clinical utility.
+
+The PsySys app represents a software application which allows users to build their perceived causal network as part of a modular learning experience. Thereby, it extends existing methods to build PCR networks in the following ways. Firstly, it is implemented in a modular way and therefore allows for a less repetitive structure. Secondly, it generates the network in the background, thereby providing direct visual feedback. These new features in combination with the educational videos, allow for a stand-alone and self-explanatory, application that could be used in both research and clinical practice. 
```

Comment:
What do you mean exactly with "learning experience"? Are they learning about what causal networks are and how they could be for themselves (but the generated network is not necessarily their actual network, but more of an example). Or is it more learning about themselves and what their actual networks are? Maybe you could specify this a bit more?